### ISSUE TITLE
generic sensor presence for wind and other data

### ISSUE THREAD

**Issue by sommo** [naofireblade/homebridge-weather-plus#97]
Don't know why, i tried few solution without success (removed persist accessories, update the plugin, restart many time the homebridge instance) but everytime when i added to homekit, wind sensor and other stuff are reported with name "sensor presence" generic so i don't know what sensor is.

my config, very simple (it works few weeks ago, disapperead from homekit, and i found that is what is happenend).

        },
        "accessories": [],
        "platforms": [{
        "platform": "WeatherPlus",
        "service": "darksky",
        "key": "xxxxxxxxxxxxxxxxxxxxx",
        "locationGeo": [44.3631682, 11.761550],
        "compatibility": "both",
        "forecast": [2],
        "conditionCategory": "detailed",
        "nameForecast": "Previsioni Imola"
    }

the homebridge run without error, but i get sensor name wrong like ai said before.
Thank you.

**Comment by sommo** [comment]
no one has this issue? for example wind sensor are named generic sensor and so on for other 3 sensor. so i can't know which sensor is exposing what.

**Comment by naofireblade** [comment]
Hi, I heard from 2 persons in a forum about the same problem. Thats something within homekit that i cannot control. Did u change anything prior to the problem?
What other devices do u have in your homekit setup? What's your configured home hub (iPad, AppleTV, HomePod)?
This might help to narrow down the problem.

**Comment by sommo** [comment]
I didn’t make change prior.
I use iPad as homehub.
If there is some debug to launch let me know.
Thank you.

Inviato da iPhone

**Comment by sommo** [comment]
i tried again but i can use this plugin, it do not report correct name for sensor

**Comment by naofireblade** [comment]
Yes the problem still exists for some. I still have no idea why :(

**Comment by dacarson** [comment]
I believe this is the same issue as #247
Once the PR #285 lands, this should be fixed.
